pages/lp/[slug].tsx

```diff
@@ -67,13 +67,21 @@ const LandingPage: NextPageWithLayout<IProps> = ({
           <div key={block.title} className="fr-col-12 fr-col-sm-4 fr-col-md-4">
             <div className="reassurance">
               <h2>{block.title}</h2>
-              <ReactMarkdown>{block.body}</ReactMarkdown>
+              <div
+                dangerouslySetInnerHTML={{
+                  __html: parseMarkdownSync(block.body).html,
+                }}
+              />{' '}
```

Comment:
You repeat several time the same block which : 
- do tricky things (`dangerouslySetInnerHtml` is a bit dangerous)
- import `parseMarkdownSync`

I would feel more comfortable with something  like : 

`<RenderMarkdown  data={block.body} />`

Where `parseMarkdownSync` is called internaly and more documented and where data is typed using `IParsedMakdown`